js/forms.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Функция возвращает callback-функцию
+ * @param {string} methodName - флаг в какое состояние переключить
+ * @returns {switchTo} callback-функция
+ */
+const switchTo = (methodName) => (child) =>
+  child[methodName]('disabled', 'disabled');
+
+/**
+ * Функция добавляет или убирает атрибут 'disabled' у дочерних элементов формы
+ * @param {HTMLElement} form - форма
+ * @param {switchTo} useMethod - callback-функция
+ */
+const switchChildren = (form, useMethod) => {
+  [...form.elements].forEach(useMethod);
+};
+
+export const disableForms = (forms) => {
+  forms.forEach((form) => {
+    form.classList.add('ad-form--disabled');
+    switchChildren(form, switchTo('setAttribute'));
+  });
+};
+
+export const enableForms = (forms) => {
+  forms.forEach((form) => {
+    form.classList.remove('ad-form--disabled');
+    switchChildren(form, switchTo('removeAttribute'));
+  });
+};
+
+export const hideNode = (node) => {
+  node.classList.add('visually-hidden');
+};
+
+export const validateAdForm = () => {
+  const form = document.querySelector('.ad-form');
+  const rooms = form.querySelector('#room_number');
+  const capacity = form.querySelector('#capacity');
+
+  const pristine = new Pristine(form, {
```

Comment:
+ тебе за поиск и использование сторонней библиотеки. это важно - делать меньше своими руками

для обозначения того, что Pristine это глобальная переменная в окне
```suggestion
  const pristine = new window.Pristine(form, {
```

Reply:
А почему [pristine] должна быть глобальной? Мне кажется, что [pristine] должна быть локальной переменной в области видимости validateAdForm(), т.к. экземпляр создается с уникальными для формы настройками (классы инпутов). Для любой другой формы нужно будет создавать  новый экземпляр [pristine] со своими настройками... В чём тонкость?

Reply:
Мне кажется, что обращение к данному экземпляру имеет смысл только внутри validateAdForm() ...

Reply:
она, конечно, в идеале, должна быть локальной. но библиотека офрмлена так, что модуль `Pristine` является глобальной переменной 
Тоноксть в том, что изначально в 1996 году никаких модулей не придумали. и в последствии пришлось много извращаться, пока в 2015 (т.е. 20 лет) сумели договориться "что такое модуль"

Reply:
модули, если интересно - погугли 
AMD, UMD, IIFE CommonJS, System - и это только часть :)